cabinet/cabinet.py

```diff
@@ -56,6 +56,20 @@ def get_all(self):
     def get(self, name):
         return self._vault.get(name)
 
+    def get_by_tags(self, tags=[]):
```

Comment:
I see that you added the conditional but you missed the parameter change (`s/=[]/=None/`)